packages/memory-adapter/modules/adapter/adapter.js

```diff
@@ -16,6 +16,7 @@ const adapterState: AdapterState = {
 
 const configure = ({
   user,
+  defaultFlags = {},
```

Comment:
The adapter receives the defaultFlags (named this way if I'd start again I'd name it initialFlags.